src/edu/csus/ecs/pc2/core/scoring/DefaultScoringAlgorithm.java

```diff
@@ -831,6 +877,46 @@ private void createStandingXML (TreeMap<StandingsRecord, StandingsRecord> treeMa
         summaryMememento.putInteger("medianProblemsSolved", getMedian(srArray));
         generateSummaryTotalsForProblem (problems, problemsIndexHash, summaryMememento);
 
+        // We do not do medal or honors citations for individual groups/sites
+        // since it does not make sense.  The values set for those are for the
+        // entire contest, not one particular group (or groups).  So, if any groups were excluded
+        // from the scoreboard, we do not fill in citations.
+        if(!excludedGroups) {
+            // Now go back and fill in the awards for each team if not for specific group
+            ResultsFile resultsInfo = new ResultsFile();
+            // Note that createCitationRankInformation() may change the order of srArray
+            CitationRankInformation ri = resultsInfo.createCitationRankInformation(theContest, srArray);
+            if(index == teamStandingsMementos.size()) {
+                int teamRank, sIndex;
+                IMemento standingsRecordMemento;
+                for(sIndex = 0; sIndex < index; ) {
+                    standingsRecordMemento = teamStandingsMementos.get(sIndex);
+                    teamRank = srArray[sIndex].getRankNumber();
+                    sIndex++;
+
+                    // For medals, we use the "place" not the "rank"
+                    // For honors we rank since multiple ranks can be part of each of the honors groups.
+                    if(ri.getFirstHonorableMentionRank() > 0 && teamRank >= ri.getFirstHonorableMentionRank()) {
+                        standingsRecordMemento.putBoolean("isHonable", true);
```

Comment:
Oops, I found another place where I would request a change before improving:  the text name of the attribute in the above line should be `isHonorable`, not `isHonable`.